source/digits_hits/src/GateRejectedVolume.cc

```diff
@@ -0,0 +1,68 @@
+/*----------------------
+   Copyright (C): OpenGATE Collaboration
+
+This software is distributed under the terms
+of the GNU Lesser General  Public Licence (LGPL)
+See LICENSE.md for further details
+----------------------*/
+
+
+#include "GateRejectedVolume.hh"
+
+#include "G4UnitsTable.hh"
+
+#include "GateRejectedVolumeMessenger.hh"
+#include "GateTools.hh"
+
+GateRejectedVolume::GateRejectedVolume(GatePulseProcessorChain* itsChain,
+			       const G4String& itsName,
+      	      	      	      	 G4String rejectedVolume)
+  : GateVPulseProcessor(itsChain,itsName),
+    m_rejectedVolume(rejectedVolume)
+{
+  m_messenger = new GateRejectedVolumeMessenger(this);
+}
+
+
+
+
+GateRejectedVolume::~GateRejectedVolume()
+{
+  delete m_messenger;
+}
+
+
+
+void GateRejectedVolume::ProcessOnePulse(const GatePulse* inputPulse,GatePulseList& outputPulseList)
+{
+  if (!inputPulse) {
+    if (nVerboseLevel>1)
+      	G4cout << "[GateRejectedVolume::ProcessOnePulse]: input pulse was null -> nothing to do\n\n";
```

Comment:
Change tab to spaces and change it 2 spaces to be consistence with whole file.

Reply:
done